### ISSUE TITLE
Lore stylesheet refers to overly specific fonts, does not display monospaced text in code samples on a default ubuntu installation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3121]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#3121|
|Type|enhancement|
|Created|2008-03-26 05:20:35Z|

The default font family should really be "monospace", which coincidentally looks better than neep anyway.

Attachments:

* [monostyle.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/215/2156c7dfc08f5a465b072928d6f26c9612612169/3bfb674d9a62fbc1e3ea42fcb59eaa314efee6ef.diff) (850 bytes) - added by glyph on 2008-09-05 15:17:34Z - 
* [default-monospace-is-fine.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/215/2156c7dfc08f5a465b072928d6f26c9612612169/247bc86b4cb1507b8fcdf1b499ddc0fa43b59f93.patch) (694 bytes) - added by glyph on 2008-11-02 16:29:37Z - Oops agttach the patch to be reviewed.

<details><summary>Searchable metadata</summary>

```
trac-id__3121 3121
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation documentation
time__1206508835000000 1206508835000000
changetime__1225816929000000 1225816929000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set owner to @exarkun|
|-|-|

Turns out 12pt is kind of huge for 'monospace'.  What do you think?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @glyph|
|-|-|

With only the font-family change, this renders exactly the same for me.  I think 9pt is pretty small.  Why do we even have a font-size at all?  I would be for dropping the font-size.  Since adding Monospace does nothing to change the display for me, I'm not going to comment on that part of the change.  What environment does this make a difference for?

Also, I can't conceive of how this might be highest priority.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph removed owner|
|-|-|

Upon further consideration, modern browsers on all platforms display monospaced fonts pretty well.  This font specification is a holdover from the days when Netscape 4 on linux looked absolutely god-awful.  There's not really any need to specify a font for `<pre>`.

I have no idea how your browser has fonts configured, but this new patch makes the fonts look considerably better on my stock Ubuntu and stock Windows-firefox browsers.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_5"></a>|@jml set owner to @glyph|
|-|-|

Looks great. Please merge.

Also, thanks for getting around to this. It's easy to let easy cosmetic improvements slide.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph set status to `closed`|
|-|-|

(In [25323]) Adjust the stylesheets on lore documents to simply use the default `PRE` font.

This change remove font and size rules left over from an earlier generation of browsers, when the defaults looked truly awful.  These rules would sometimes cause code samples to be displayed in a proportional font, so this corrects buggy rendering of the documentation in some configurations.

Author: glyph

Reviewers: exarkun, jml

Fixes [#3121](https://github.com/twisted/twisted/issues/3121)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
